### ISSUE TITLE
Problem accessing Vicki skill

### ISSUE THREAD

**Issue by ACprof** [heyvicki/vicki-browser-ext#2]
Hi.

I'm having a problem with your alexa skill "Vicki | control my computer".
I followed the steps from your site "http://heyvicki.net/", but when I say any command to Alexa, like "Alexa, ask web browser to browse amazon", she replies by saying that she's having trouble accessing Vicki skill.

Do you know what might be causing this?

Thank you.

**Comment by paveljamal** [comment]
If others are having similar issue, do comment here in case it is a common issue.

**Comment by Wua4** [comment]
Is Vicki still working? Can't get it to access the skill.

**Comment by karup** [comment]
I can find the skill in .com but My account is .ca and there is no vicki skill

**Comment by Gabrielhj17** [comment]
@karup i have the same problem, my account is .co.uk but i can only find the skill on the .com page

**Comment by dotslash403** [comment]
Same issue in Canada .ca

**Comment by princeofmysteryworld** [comment]
> If others are having similar issue, do comment here in case it is a common issue.

yes this a common issue....I AM FACING THIS SAME ISSUE...plz fix this

**Comment by hrgog** [comment]
same problem here...   any help yet?

**Comment by joejoe902** [comment]
same problem

**Comment by avi4zero** [comment]
same problem here , I think it is not working now
